@types/jsb.d.ts

```diff
@@ -193,29 +193,47 @@ declare namespace jsb {
          * ```ts
          * // create a download task
          * let downloader = new jsb.Downloader();
-         * let task = downloader.createDownloadTask('https://example.com/exampleFile.zip', jsb.fileUtils.getWritablePath());
-         * down.setOnTaskProgress((task, recv, total)=>{ console.log(recv, total);})
+         * let task = downloader.createDownloadFileTask('https://example.com/exampleFile.zip', jsb.fileUtils.getWritablePath());
          */
         createDownloadFileTask (requestURL:string, storagePath:string, identifier?:string): DownloaderTask;
```

Comment:
Can remove Downloader related docs here, as it is moved to `cocos/native-binding/index.ts`.

Reply:
jsb.d.ts just keep the API that are not used by developers.

Reply:
if the API exported to user, it should put in the native-bindinng module

if the API is only for engine dev, it should put in the jsb.d.ts

the declaration should be merged together in the future, here is the trackinng issue https://github.com/cocos/3d-tasks/issues/12609

Reply:
Already removed.

Reply:
> if the API exported to user, it should put in the native-bindinng module
> 
> if the API is only for engine dev, it should put in the jsb.d.ts
> 
> the declaration should be merged together in the future, here is the trackinng issue [cocos/3d-tasks#12609](https://github.com/cocos/3d-tasks/issues/12609)

got it.